packages/core/test/ArrayField_test.js

```diff
@@ -2192,7 +2192,7 @@ describe("ArrayField", () => {
         },
       ],
     };
-    it("should pass form context to schema field", () => {
+    it("should pass form context to array schema field only", () => {
```

Comment:
Shouldn't this be an additional test, rather than changing the existing test?

Reply:
Good point, I'll add that